plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/glue/DeltaLakeGlueMetastoreTableOperations.java

```diff
@@ -0,0 +1,110 @@
+import software.amazon.awssdk.services.glue.model.EntityNotFoundException;
+import software.amazon.awssdk.services.glue.model.GetTableRequest;
+import software.amazon.awssdk.services.glue.model.Table;
+import software.amazon.awssdk.services.glue.model.TableInput;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static io.trino.plugin.deltalake.metastore.DeltaLakeTableMetadataScheduler.deltaParameters;
+import static java.util.Objects.requireNonNull;
+
+public class DeltaLakeGlueMetastoreTableOperations
+        extends DeltaLakeTableOperations
+{
+    private final GlueClient glueClient;
+    private final GlueContext glueContext;
+    private final GlueCache glueCache;
+    private final GlueMetastoreStats stats;
+
+    @Nullable
+    private String glueVersionId;
+
+    public DeltaLakeGlueMetastoreTableOperations(
+            ConnectorSession session,
+            GlueClient glueClient,
+            GlueContext glueContext,
+            GlueCache glueCache,
+            GlueMetastoreStats stats,
+            SchemaTableName schemaTableName)
+    {
+        super(session, schemaTableName);
+        this.glueClient = requireNonNull(glueClient, "glueClient is null");
+        this.glueContext = requireNonNull(glueContext, "glueContext is null");
+        this.glueCache = requireNonNull(glueCache, "glueCache is null");
+        this.stats = requireNonNull(stats, "stats is null");
+    }
+
+    @Override
+    public void commitToExistingTable(long version, String schemaString, Optional<String> tableComment)
+    {
+        GetTableRequest getTableRequest = GetTableRequest.builder()
+                .databaseName(schemaTableName.getSchemaName())
+                .name(schemaTableName.getTableName())
+                .build();
+        Table currentTable;
+        try {
+            currentTable = glueClient.getTable(getTableRequest).table();
+        }
+        catch (EntityNotFoundException e) {
+            throw new TableNotFoundException(schemaTableName);
+        }
+        glueVersionId = currentTable.versionId();
+
+        stats.getUpdateTable().call(() -> glueClient.updateTable(builder -> builder
+                .applyMutation(glueContext::configureClient)
+                .databaseName(schemaTableName.getSchemaName())
+                .tableInput(convertGlueTableToTableInput(currentTable, version, schemaString, tableComment))
+                .versionId(glueVersionId)));
+        glueCache.invalidateTable(schemaTableName.getSchemaName(), schemaTableName.getTableName(), false);
+    }
+
+    private static TableInput convertGlueTableToTableInput(Table glueTable, long version, String schemaString, Optional<String> tableComment)
+    {
+        Map<String, String> parameters = ImmutableMap.<String, String>builder()
+                .putAll(glueTable.parameters())
+                .putAll(deltaParameters(version, schemaString, tableComment))
+                .buildKeepingLast();
+
+        return TableInput.builder()
```

Comment:
should we reuse below like you did it for v1/GlueHiveMetastore? by overriding parameters from the returned builder?

https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java#L673-L686